src/main/java/com/yurjinia/project_structure/project/service/ProjectService.java

```diff
@@ -2,91 +2,194 @@
 
 import com.yurjinia.common.exception.CommonException;
 import com.yurjinia.common.exception.ErrorCode;
+import com.yurjinia.project_structure.project.dto.CreateProjectRequest;
+import com.yurjinia.project_structure.project.dto.InviteToProjectRequest;
 import com.yurjinia.project_structure.project.dto.ProjectDTO;
+import com.yurjinia.project_structure.project.dto.UpdateProjectRequest;
 import com.yurjinia.project_structure.project.entity.ProjectEntity;
 import com.yurjinia.project_structure.project.repository.ProjectRepository;
 import com.yurjinia.project_structure.project.service.mapper.ProjectMapper;
+import com.yurjinia.user.dto.UserDTO;
 import com.yurjinia.user.entity.UserEntity;
 import com.yurjinia.user.service.UserService;
+import com.yurjinia.user.service.mapper.UserMapper;
 import jakarta.transaction.Transactional;
 import lombok.RequiredArgsConstructor;
+import org.apache.commons.lang3.StringUtils;
 import org.springframework.http.HttpStatus;
 import org.springframework.stereotype.Service;
 
 import java.util.List;
 import java.util.Set;
 import java.util.stream.Collectors;
 
+import static com.yurjinia.common.exception.ErrorCode.PROJECT_NOT_FOUND;
+
 @Service
 @RequiredArgsConstructor
 public class ProjectService {
 
     private final UserService userService;
     private final ProjectMapper projectMapper;
     private final ProjectRepository projectRepository;
+    private final UserMapper userMapper;
```

Comment:
move UserMapper to UserService, all places where you call userMapper - create a method in user service and map Set<UserEntity> to List<UserDTO>.